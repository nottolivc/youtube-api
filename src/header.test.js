import { render } from '@testing-library/react';
//import { Header } from './components/Header';

jest.mock('react-router-dom');
describe('<Header />', () => {

  it('renders the basic <Header />', () => {
    const { container } = render('<Header />');
    expect(container.firstChild).toMatchSnapshot();
  });
});