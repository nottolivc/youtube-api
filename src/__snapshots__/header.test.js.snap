// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders the basic <Header /> 1`] = `&lt;Header /&gt;`;
